server/integration/endpoint/src/main/java/org/infinispan/server/endpoint/subsystem/CacheIgnoreStatusHandler.java

```diff
@@ -0,0 +1,47 @@
+package org.infinispan.server.endpoint.subsystem;
+
+import org.jboss.as.controller.OperationContext;
+import org.jboss.as.controller.OperationFailedException;
+import org.jboss.as.controller.OperationStepHandler;
+import org.jboss.as.controller.PathAddress;
+import org.jboss.as.controller.registry.Resource;
+import org.jboss.as.controller.registry.Resource.ResourceEntry;
+import org.jboss.dmr.ModelNode;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.infinispan.server.endpoint.subsystem.ModelNodeUtils.contains;
+
+/**
+ * @author gustavonalle
+ * @since 9.0
+ */
+public class CacheIgnoreStatusHandler implements OperationStepHandler {
+
+   public static final CacheIgnoreStatusHandler INSTANCE = new CacheIgnoreStatusHandler();
```

Comment:
I never was sure why you'd need to "enforce" the singleton pattern, but if you do, I'd much rather make the default constructor private than make the class an enum.